### PR TITLE
[sailfish-browser] Remove separate .desktop files for MIME type handling. OMP#JOLLA-138 JB#54217

### PR DESCRIPTION
**Pull request by adenexter**
A single file per application makes it possible to inspect the
permissions of the application handling a MIME type without having to do
a secondary resolution to find the file with the permissions.

### CODE REVIEW

File: sailfish-browser.desktop [hunk 1]
```diff
@@ -6,6 +6,10 @@ X-MeeGo-Translation-Catalog=sailfish-browser
 Icon=icon-launcher-browser
 Exec=/usr/bin/sailjail -p sailfish-browser.desktop /usr/bin/sailfish-browser %U
 Comment=Sailfish UI application
+MimeType=text/html;x-scheme-handler/http;x-scheme-handler/https;
```
Comment (okodron) [inline sailfish-browser.desktop:9]: Why have you dropped `text/xml;application/xhtml+xml;application/xml`?

Comment (adenexter) [reply sailfish-browser.desktop:9]: I do not know. I thought I'd just copied as if from the removed .desktop file.

Comment (okodron) [reply sailfish-browser.desktop]: Got it.

**Comment by rainemak** [comment]
After reporting some other findings I can approve this one.

**Review by rainemak — APPROVED** [review]
Please rebase before merging.

**Comment by rainemak** [comment]
@dseight or @okodron review

**Comment by rainemak** [comment]
> @dseight or @okodron review

Ping

**Review by dseight — APPROVED** [review]

**Review by okodron — APPROVED** [review]

**Pull request merged**